models/cxr_image_synthesis_latent_diffusion_model/configs/metadata.json

```
@@ -0,0 +1,73 @@
+{
+    "schema": "https://github.com/Project-MONAI/MONAI-extra-test-data/releases/download/0.8.1/meta_schema_20220324.json",
```

```suggestion
    "schema": "https://github.com/Project-MONAI/MONAI-extra-test-data/releases/download/0.8.1/meta_schema_20240725.json",
```
Let's try the current schema to see if this resolves the issues about the network arguments.